.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         toolchain: [
           stable,
           beta,
-          1.56.0 # MSRV
+          1.60.0 # MSRV
```

Comment:
Out of interest, what is the reason for this bump?

Reply:
The `dep:` syntax! It was only stabilised in 1.60.0.

Reply:
Ah, I see